Source/LightboxViewCell.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+public class LightboxViewCell: UICollectionViewCell {
+
+  public static let reuseIdentifier: String = "LightboxViewCell"
+
+  var lightboxView: LightboxView?
+
+  public func configureCell(image: UIImage) {
+    backgroundColor = .redColor()
+
+    if let lightboxView = lightboxView where lightboxView.superview != nil  {
```

Comment:
I think, it would be better, if the LightBoxView initialization process takes place only once, during the cell initialization and then, when the cell is being reused only the image of LightBoxView is being swapped, but no views are being added/removed from the Cell.

With this approach we can avoid:
```Objective-C
lightboxView.removeFromSuperview()
```
```Objective-C
contentView.addSubview(lightboxView!)
```
that are main-thread UI-related operations and potentially can become a bottleneck and call initialization code "lazily"
```Objective-C
        lightboxView = LightboxView(frame: bounds,
            image: image)
```
This can improve performance